rviz_plugin/slam_toolbox_rviz_plugin.cpp

```diff
@@ -556,18 +556,27 @@ void SlamToolboxPlugin::updateCheckStateIfExternalChange()
 
   while (rclcpp::ok()) {
     auto parameters = parameters_client->get_parameters(
-      {"paused_new_measurements", "interactive_mode"});
-    paused_measure = parameters[0].as_bool();
-    interactive = parameters[1].as_bool();
-
-    bool oldState = _check1->blockSignals(true);
-    _check1->setChecked(interactive);
-    _check1->blockSignals(oldState);
-
-    oldState = _check2->blockSignals(true);
-    _check2->setChecked(!paused_measure);
-    _check2->blockSignals(oldState);
-
+      {"paused_new_measurements", "interactive_mode"}, std::chrono::seconds(1));
+    if (parameters.empty()) {
+      RCLCPP_INFO_ONCE(
```

Comment:
I think this should be continuous. If there's an error (i.e. can't get it) it would be good to know

Reply:
fixed [1cd071a9ae2baee3c1f683a6d0be878c3ac5181c]

```console
[INFO] [1704336526.145135604] [rviz2]: Stereo is NOT SUPPORTED
[INFO] [1704336526.145200634] [rviz2]: OpenGl version: 4.5 (GLSL 4.5)
[INFO] [1704336526.185517412] [rviz2]: Stereo is NOT SUPPORTED
[INFO] [1704336527.251688731] [SlamToolboxPlugin]: Waiting for the slam_toolbox node configuration.
[INFO] [1704336532.253298384] [SlamToolboxPlugin]: Waiting for the slam_toolbox node configuration.
[INFO] [1704336537.254611749] [SlamToolboxPlugin]: Waiting for the slam_toolbox node configuration.
[INFO] [1704336540.256296180] [SlamToolboxPlugin]: Start the slam_toolbox node state check.  ★ start slam_toolbox
[INFO] [1704336553.247976056] [SlamToolboxPlugin]: Waiting for the slam_toolbox node configuration.  ★ kill slam_toolbox
[INFO] [1704336558.249417605] [SlamToolboxPlugin]: Waiting for the slam_toolbox node configuration.
[INFO] [1704336561.250983968] [SlamToolboxPlugin]: Start the slam_toolbox node state check. ★ restart slam_toolbox
```